client/posts/view.ts

```diff
@@ -245,20 +252,85 @@ export default class PostView extends ImageHandler {
         el.innerHTML = html
     }
 
-    // Render "USER WAS BANNED FOR THIS POST" message
-    public renderBanned() {
+    // Render related mod-log status
+    public async renderStatus(type: number) {
+        var message: string
+        var after: boolean
         this.uncheckModerationBox()
-
-        const el = firstChild(this.el.querySelector(".post-container"), el =>
-            el.classList.contains("banned"))
-        if (el) {
+        const el = this.el.querySelector(".post-container")
+
+        if (
+            firstChild(el, el => {
+                after = el.classList.contains("banned")
```

Comment:
So that we don't add the same message twice.
I'm not sure rebuilding is the best idea in this case, since we have to check anyway.
We could select all `<b>` tags and then delete and then rebuild, but we would still have to check if it contained "BANNED" or "DELETED" or "VIEWED", so I'm just not sure.
Do you have an idea of might be better while maintaining this capability?

Reply:
Just add some class to these and remove all children of this class (if any).

Reply:
Okay.